src/app/components/card/card.tsx

```diff
@@ -0,0 +1,21 @@
+import { Typography } from '@mui/material';
+import React from 'react'
+import './card.css'
+
+function Card(props: any): JSX.Element {
+  const imageApiUrl = "https://image.tmdb.org/t/p/w500";
```

Comment:
Move this to some separate `constants.ts` file.
Or move from the component at least.